thinkhazard/views/report.py

```diff
@@ -64,4 +68,5 @@ def report(request):
     hazard_data = hazard_data.values()
     hazard_data = sorted(hazard_data, key=lambda d: d['categorytype'].order)
 
-    return {'hazards': hazard_data, 'division': division}
+    return {'hazards': hazard_data, 'division': division,
+            'parent_division': division.parent}
```

Comment:
You're right. I personally like this way because it justifies the use of a `join` in the SQL query.